### ISSUE TITLE
Night Mode

### ISSUE THREAD

**Issue by autofreak** [pmiddend/piggybudget#6]
I love simplicity of this app. Kindly include night mode options for users with bad eyes.

**Comment by pmiddend** [comment]
Good idea! Do you want this implemented when the sun sets (might be difficult to tell) or generally? I'm thinking about a button in the settings tab currently.

**Comment by Keplyx** [comment]
In my opinion a simple button is the best option, and the most common one. People who use nightmode usually have it always active.

**Comment by autofreak** [comment]
I want to be able to activate it at will, not necessarily automated.
